src/adBreak.js

```diff
@@ -61,14 +64,19 @@ function end(player) {
     player.addClass('vjs-live');
   }
   if (!player.ads.shouldPlayContentBehindAd(player)) {
-    snapshot.restorePlayerSnapshot(player, player.ads.snapshot);
+    snapshot.restorePlayerSnapshot(player, player.ads.snapshot, callback);
   }
 
   // Reset the volume to pre-ad levels
   if (player.ads.shouldPlayContentBehindAd(player)) {
     player.volume(player.ads.preAdVolume_);
   }
 
+  // We're done because we're not restoring a snapshot
+  if (player.ads.shouldPlayContentBehindAd(player)) {
```

Comment:
Why not include this in the previous if?

Reply:
Doing that